src/Spryker/Zed/CodeGenerator/Business/Generator/Yves/Controller/YvesControllerProviderCodeGenerator.php

```diff
@@ -7,11 +7,37 @@
 
 namespace Spryker\Zed\CodeGenerator\Business\Generator\Yves\Controller;
 
+use Spryker\Zed\CodeGenerator\Business\Engine\GeneratorEngineInterface;
 use Spryker\Zed\CodeGenerator\Business\Generator\Yves\AbstractYvesControllerCodeGenerator;
 use Zend\Filter\Word\CamelCaseToDash;
 
 class YvesControllerProviderCodeGenerator extends AbstractYvesControllerCodeGenerator
 {
+    /**
+     * @var string
+     */
+    protected $providerNameSpace;
+
+    /**
+     * YvesControllerProviderCodeGenerator constructor.
+     *
+     * @param string $bundle
+     * @param \Spryker\Zed\CodeGenerator\Business\Engine\GeneratorEngineInterface $generatorEngine
+     * @param string $controller
+     * @param string $providerNameSpace
+     * @param array $requiredGenerators
+     */
+    public function __construct(
+        string $bundle,
```

Comment:
module !

Reply:
Old code can also be adjusted to the proper variable name as long as it doesnt break anything (and remains BC).
So inside methods the naming scope is usually fine to be fixed up as well.